includes/class-sensei-admin.php

```diff
@@ -215,8 +215,11 @@ function install_pages_output() {
 			?>
 			<div id="message" class="updated sensei-message sensei-connect">
 				<p><?php echo wp_kses_post( __( '<strong>Congratulations!</strong> &#8211; Sensei LMS has been installed and set up.', 'sensei-lms' ) ); ?></p>
-				<p><a href="https://twitter.com/share" class="twitter-share-button" data-url="https://woocommerce.com/products/sensei/" data-text="A premium Learning Management plugin for #WordPress that helps you create courses. Beautifully." data-via="senseilms" data-size="large" data-hashtags="Sensei">Tweet</a>
-				<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src="//platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script></p>
+				<p>
+					<a href="<?php echo esc_url( admin_url( 'post-new.php?post_type=course' ) ); ?>" class="button-primary">
+						<?php esc_html_e( 'Create Course', 'sensei-lms' ); ?>
```

Comment:
`Create a Course`? `Create your first Course`? 🤔

Reply:
I was going back and forth between all of those... sometimes with exclamation marks. I'll try `Create a Course`. e12ce352fa361b72c81664ed4857b5f201d3d861

Reply:
I like more "Create your first Course", but I also like "Create a Course" =)